drivers/rkt/driver_test.go

```diff
@@ -412,23 +410,22 @@ func TestRktDriver_StartWaitRecoverWaitStop(t *testing.T) {
 	ch, err = harness.WaitTask(context.Background(), task.ID)
 	require.NoError(err)
 
-	wg.Add(1)
-	waitDone = false
 	go func() {
-		defer wg.Done()
-		result := <-ch
+		time.Sleep(300 * time.Millisecond)
```

Comment:
Hopefully we can remove this sleep? If InspectTask can see a task then StopTask should be able to work without a sleep. Not sure why the original test had it. Perhaps ported from a time when such guarantees did not exist?

Reply:
removed it and it worked quite well.